tools/js-tools/types/global.d.ts

```diff
@@ -0,0 +1,108 @@
+declare module '*.mdx';
+declare module '*.module.scss' {
+	const classes: { [ key: string ]: string };
+	export default classes;
+}
+
+type AvailableBlockProps =
+	| {
+			available?: boolean;
+	  }
+	| undefined;
+
+interface Window {
+	Initial_State?: {
+		adminUrl?: string;
+	};
+	JP_CONNECTION_INITIAL_STATE: {
```

Comment:
I did a grep for this identifier, and turned up mentions in global.d.ts files in packages/my-jetpack and packages/videopress.

Looks like both of those aren't having any typescript type checking done at all. A followup PR might either add a run of `tsc` somewhere in the build or test scripts for those too, or add `ForkTSCheckerPlugin: { typescript: { mode: 'readonly' } }` in the webpack config.